docs/src/Upsell.doc.js

```diff
@@ -22,174 +41,412 @@ card(
+    }}
+  />
+  <Upsell
+    imageData={{
+      component: <Icon icon="send" accessibilityLabel="" color="darkGray" size={32} />,
+    }}
+    title="So close! Finish installing your pinterest tag, get $10 in ads credit"
+    message="Track ads conversion—sales, traffic and more—with the Pinterest tag"
+    primaryAction={{ label: 'Claim now' }}
+  />
+</Flex>;
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+          Use Upsells for critical information, such as errors or warnings. Use [Callout](/Callout) instead. Upsells should not be used for general information either.
+        `}
+        defaultCode={`
 <Upsell
-  message="Single line upsell with no title or call to action."
+  title="Could not link account"
+  message="There was a problem connecting your account."
+  primaryAction={{ label: 'Try again' }}
   dismissButton={{
     accessibilityLabel: 'Dismiss banner',
-    onDismiss: ()=>{},
+    onDismiss: () => {},
   }}
-/>
-  `}
-  />,
+  imageData={{
+    component: <Icon icon="warning" accessibilityLabel="Warning" color="darkGray" size={32} />,
+  }}
+/>;
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        Stack Upsells on a page. In the case that they must be stacked, [Callouts](/Callout) will appear above Upsells.
+        `}
+        defaultCode={`
+<Box>
+  <Box marginBottom={4} display="flex" alignItems="center">
+    <Icon accessibilityLabel="" icon="pinterest" color="red" size={32} />
+    <ButtonGroup>
+      <Button color="transparent" iconEnd="arrow-down" text="Business" inline />
+      <Button color="transparent" iconEnd="arrow-down" text="Create" inline />
+      <Button color="transparent" iconEnd="arrow-down" text="Analytics" inline />
+      <Button color="transparent" iconEnd="arrow-down" text="Ads" inline />
+    </ButtonGroup>
+  </Box>
+  <Divider />
+  <Box marginTop={8}>
+    <Flex gap={2} direction="column">
+      <Upsell
+        imageData={{
+          component: <Icon icon="send" accessibilityLabel="" color="darkGray" size={32} />,
+        }}
+        title="So close! Finish installing your pinterest tag, get $10 in ads credit"
+        message="Track ads conversion—sales, traffic and more—with the Pinterest tag"
+        primaryAction={{ label: 'Claim now' }}
+      />
+      <Upsell
+        title="Give $30, get $60 in ads credit"
+        message="Earn $60 of ads credit, and give $30 of ads credit to a friend"
+        primaryAction={{ href: 'https://pinterest.com', label: 'Send invite', target: 'blank' }}
+        dismissButton={{
+          accessibilityLabel: 'Dismiss banner',
+          onDismiss: () => {},
+        }}
+        imageData={{
+          component: <Icon icon="pinterest" accessibilityLabel="" color="darkGray" size={32} />,
+        }}
+      />
+    </Flex>
+  </Box>
+</Box>;
+        `}
+      />
+      <MainSection.Card
```

Comment:
@ayeshakmaz I'm a little bit confused as this example is not showing on the Docs....

Reply:
![Uploading image.png…]()